### ISSUE TITLE
[FEAT] Sort channels in alphabetically

### ISSUE THREAD

**Issue by vikasganiga05** [collab-community/coding-youtubers#4]
If we sort the channels in alphabetical order. The users can easily find them. Also, the merge conflicts will be reduced.

**Comment by KendallDoesCoding** [comment]
Good idea, can we find a gh-action or something to do this, or will we have to do it manually when user makes a PR to add the channel?

**Comment by KendallDoesCoding** [comment]
The problem with this is that, the list will take lots of space where as the user will have to scroll quite a bit down, do you get what I mean @vikasganiga05?

**Comment by vikasganiga05** [comment]
> The problem with this is that, the list will take lots of space where as the user will have to scroll quite a bit down, do you get what I mean @vikasganiga05?

Yes, I got your point. It's your choice. We can mention in the readme to use `Ctrl + F` to find the channels by keywords.

**Comment by KendallDoesCoding** [comment]
Good idea, let's proceed with this in that case.

**Comment by KendallDoesCoding** [comment]
@vikasganiga05 can you work on this?

**Comment by Preet-Sojitra** [comment]
How about sorting on basis of content?
I mean separate table for web dev content, separate for DSA, python, etc etc.... and if channel has on various topics it can be added under "miscellaneous" section. 
Is it good idea?

Since right now there are less channel in markdown, so sorting will be easier, if more people start contributing and we haven't added any instruction about how to add new youtube channels, then it will be difficult to sort afterwords

**Comment by KendallDoesCoding** [comment]
> How about sorting on basis of content? I mean separate table for web dev content, separate for DSA, python, etc etc.... and if channel has on various topics it can be added under "miscellaneous" section. Is it good idea?
> 
> Since right now there are less channel in markdown, so sorting will be easier, if more people start contributing and we haven't added any instruction about how to add new youtube channels, then it will be difficult to sort afterwords

Good idea.

What do you think of this @vikasganiga05 , @collab-community/coding-youtubers?

**Comment by Agrim-Sharma174** [comment]
OR... adding to that...
The table sections can be divided by web dev, DSA, etc, and also for particular programming languages

**Comment by Agrim-Sharma174** [comment]
There are links to a particular language: English, Hindi, Bengali, etc at the starting of the webpage which takes the user to that language section, then at that point, another list of links for programming languages or tech... where youtube channels are listed...(one channel can be present in multiple programming languages- I think this would be better as compared to miscellaneous)

This can save time in finding the exact channel that the user wants

How's this? @vikasganiga05 @collab-community/coding-youtubers

**Comment by vikasganiga05** [comment]
> 

This is a great idea, but the contributor should add one YT channel name in different sections to which it belongs. Let's see what other thoughts on this.

**Comment by yugborana** [comment]
> > 
> 
> This is a great idea, but the contributor should add one YT channel name in different sections to which it belongs. Let's see what other thoughts on this.

yeah this is fine.
